docs/can-guides/contribute/code.md

```diff
@@ -68,151 +68,152 @@ Download the Firefox browser
+|   ├── test.html           — Main test page
 ```
 
-Generally, speaking, the most important files are:
+Generally speaking, the most important files are:
 
- - the main module -  `can-example.js`
- - the main test module - `test/can-example-test.js`
- - the test page - `test/test.html`
+ - the main module —  `can-example.js`
+ - the main test module — `test/can-example-test.js`
+ - the test page — `test/test.html`
 
-When fixing a bug or making a feature, we'll add a test in the main test module
-and update code in the main module and verify the tests are passing by running
+To fix a bug or making a feature, add a test in the main test module, update code in the main module, and then verify the tests are passing by running
 the test page.
 
 Some modules have multiple modules, test modules, and test pages.  These modules are
-commonly organized as __modlets__ where each folder will have its own main module, test module
+commonly organized as __modlets__ where each folder will have its own main module, test module,
 and test page:
 
 ```
-├── a-module/            - Module's modlet folder
-|   ├── a-module.js      - The module
-|   ├── a-module-test.js - The module's tests
-|   ├── test.html        - A test page that runs just the module's tests
+├── a-module/            — Module’s modlet folder
+|   ├── a-module.js      — The module
+|   ├── a-module-test.js — The module’s tests
+|   ├── test.html        — A test page that runs just the module’s tests
 ```
 
-Where possible, CanJS code is:
+Where possible, CanJS code uses:
 
 - Tabs not spaces
-- JSHinted
+- JSHint
 - CommonJS not ES6
-- Follows jQuery [coding conventions](https://contribute.jquery.org/style-guide/js/)
+- jQuery’s [coding conventions](https://contribute.jquery.org/style-guide/js/)
 
 
 ##  Make your changes
 
-Once you've figured out where you need to make changes, you'll want to complete the following steps
+Once you’ve figured out where you need to make changes, you’ll want to complete the following steps
 to make those changes and create a pull request so we can include your code in future releases:
 
 
 1. Create a new feature branch. For example, `git checkout -b html5-fix`.
 2. Make some changes to the code and tests.
-4. Run tests `npm test` and make sure they pass in all browsers.
-5. Update documentation if necessary.
+4. Run `npm test` to make sure the tests pass in all browsers.
+5. Update the documentation if necessary.
 6. Push your changes to your remote branch.  For example, `git push origin html5-fix`.
-7. Submit a pull request! Navigate to Pull Requests and click the 'New Pull Request' button. Fill in some
-   details about your potential patch including a meaningful title. When finished, press "Send pull request". The core team will be notified about your submission and let you know of any problems or targeted release date.
+7. Submit a pull request! On GitHub, navigate to Pull Requests and click the “New Pull Request” button. Fill in some
```

Comment:
Hey @justinbmeyer, a few comments on using the correct punctuation marks:
- It’s [pretty easy](http://www.danandcheryl.com/2010/08/how-to-type-curly-quotes-in-mac-os-x) to type the correct marks (although I don’t expect anyone else to care enough to change their habits).
- The correct marks are easier to read and can help make text more understandable.
- I’ve definitely seen issues with software automatically changing “dumb” quotes to “smart” quotes (ugh presentation software), so I wouldn’t really recommend automated replacement. Using the correct marks can actually fix some issues though, like in this PR:
<img width="730" alt="before" src="https://cloud.githubusercontent.com/assets/10070176/20915750/57ffba34-bb4d-11e6-8aa0-1a4723308085.png">
<img width="733" alt="after" src="https://cloud.githubusercontent.com/assets/10070176/20915749/57fecdf4-bb4d-11e6-839a-ff695d6f204d.png">

Again, I’m not really expecting anyone else to care, but as I go through the docs, I’m going to clean up our spelling, grammar, typography, etc. It doesn’t take much time to get the little details right. 😊